docs/provider/daemon/getting_started/run_remote.md

```diff
@@ -77,6 +77,31 @@ For this, you must obtain a valid TLS certificate from a trusted Certificate Aut
     ghcr.io/tdex-network/tdexd:latest
     ```
 
+:::tip
+If your daemon remote host uses a linux system, instead of enabling TLS on the Trade interface, you can alternatively run a tor hidden service as a proxy in front of it.
```

Comment:
I dont think should be a tip, rather that an addition "branch" of the documentation as alternative to the TLS with CA.

We should follow the same hierarchy of the tdex-box readme imho

Reply:
It shoudl go under "Enable secure connection", where we branch "either TLS CA or onion"